primer/index.bs

```diff
@@ -191,43 +192,22 @@ Response Body:
     "registration_endpoint": "https://secureauth.example/register",
     "end_session_endpoint": "https://secureauth.example/endsession",
     "jwks_uri": "https://secureauth.example/jwks",
-    "response_types_supported": [
-        "code"
-    ],
-    "grant_types_supported": [
-        "authorization_code",
-        "refresh_token"
-    ],
-    "subject_types_supported": [
-        "public"
-    ],
-    "claims_supported": [
-        "sub",
-        "webid"
-    ],
+    "response_types_supported": ["code"],
+    "grant_types_supported": ["authorization_code", "refresh_token"],
+    "subject_types_supported": ["public"],
+    "claims_supported": ["sub", "webid"],
     "scopes_supported": [
         "openid",
         "webid",
         "profile",
         "email",
         "offline_access"
     ],
-    "token_endpoint_auth_methods_supported": [
-        "client_secret_basic"
-    ],
-    "token_endpoint_auth_signing_alg_values_supported": [
-        "ES256"
-    ],
-    "request_object_signing_alg_values_supported": [
-        "ES256"
-    ],
-    "id_token_signing_alg_values_supported": [
-        "ES256"
-    ],
-    "code_challenge_methods_supported": [
-        "plain",
-        "S256"
-    ],
+    "token_endpoint_auth_methods_supported": ["client_secret_basic"],
+    "token_endpoint_auth_signing_alg_values_supported": ["ES256"],
+    "request_object_signing_alg_values_supported": ["ES256"],
+    "id_token_signing_alg_values_supported": ["ES256"],
```

Comment:
This would technically need to be `["ES256", "HS256"]` to support the example ID Token below, which is now using HS256, instead of ES256.

Reply:
We should not include HS256 in examples

Reply:
I'd normally agree HS256 isn't correct for usage here, but in lieu of having an example public/private key in repo such that the JWTs can be verified/encoded, there's not really another option.

Reply:
The problem with HS256 is that implementers look at spec examples to base their code on. Having someone base an implementation on this example would be devastating. That is a far worse thing than providing examples for which JWT.io cannot verify a signature.

Signature validation of the examples is, simply put, a separate issue and is unrelated to the point of this PR (i.e. changing WebID references to Client ID). There are many ways to address the signature validation issue without resorting to HS256